test/parallel_api/algorithm/alg.merge/inplace_merge.pass.cpp

```diff
@@ -82,6 +107,34 @@ test_by_type(Generator1 generator1, Generator2 generator2, Compare comp)
     }
 }
 
+template <typename T, typename Generator1, typename Generator2>
+void
+test_by_type(Generator1 generator1, Generator2 generator2)
```

Comment:
Could you please find a way to combine this one with another `test_by_type` to reduce code duplication?

Reply:
Done